services/database-analyzer.js

```diff
@@ -49,6 +49,12 @@ function DatabaseAnalyzer(databaseConnection, config) {
     return alias;
   }
 
+  // NOTICE: Look for the id column in both fields and primary keys
+  function hasIdColumn(fields, primaryKeys) {
+    return fields.some(field => field.name === 'id' || field.nameColumn === 'id')
+      || _.includes(primaryKeys, 'id');
```

Comment:
Not sure this is the "perfect" criteria to set the `removeAttribute('id')` in the model.